### ISSUE TITLE
Setting NavigationView `enabled` to false does not disables actions and pages

### ISSUE THREAD

**Issue by maoamid** [eclipsesource/tabris-js#1908]
### Problem description
what `enabled` propertie does on `NavigationView`?
Please ignore this, if I missing something.

### Expected behavior

?! I propose to prevent interaction with `NavigationView.toolbar` content like `actions`

### Environment

- Tabris.js version: 3.0.1
- OS: Android 7.1

### Code snippet

```javascript
import {Action, NavigationView, contentView, device, AlertDialog} from 'tabris';

const IMAGE = device.platform === 'iOS'
  ? 'resources/share-black-24dp@3x.png'
  : 'resources/share-white-24dp@3x.png';

contentView.append(
  <NavigationView stretch background='#e7e7e7' enabled={false}>
    <Action title='Action' image={IMAGE}
        onSelect={() => AlertDialog.open('Action selected.')}/>
  </NavigationView>
);
```

**Comment by mpost** [comment]
I can confirm the issue on Android and iOS. Setting the `NavigationView` to disabled should disable the pages as well as the actions.

**Comment by mpost** [comment]
The `TabFolder` has a similar behavior.

**Comment by patrykmol** [comment]
iOS is fixed. Reopening since Android part is not yet fixed

**Comment by mpost** [comment]
We have fixed the issue for the 3.2 release.
